week-2/Dela Cruz - Week 2/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+
+<head>

```

Comment:
Since the `<head>` and the `<body>` are within `<html>`, they should be indented one step in